packages/performance-tests/test/cases/identity-transformer.ts

```diff
@@ -66,11 +66,12 @@ export default async function identityTransformer(iModel: TestIModel, reporter:
       "Id": iModel.iModelId,
       "T-shirt size": iModel.tShirtSize,
       "Gb size": sizeInGb,
+      "Branch Name": branchName,
       /* eslint-enable @typescript-eslint/naming-convention */
     };
     reporter.addEntry(
       "identity transform (provenance)",
-      iModel.name,
+      `${iModel.name} - ${branchName}`,
```

Comment:
can `branchName` come first? Feels a bit less ambiguous to prefix rather than suffix, I imagine there are many iModels using hyphens. Perhaps we should prefer a colon. For example:

`federation-guid-optiization: Highway Restoration XZT-309 - Display Model`

Reply:
done